config/symbols.us.weapon.txt

```diff
@@ -1,2 +1,2 @@
-g_Weapon = 0x8017A000;
+g_unkGraphicsStruct = 0x80097400;g_Weapon = 0x8017A000;
```

Comment:
`\n` xD

Reply:
The interesting thing is that I think this problem was created by `make format`, since I had the graphics struct as the last line of this file, and `make format` sorted it. I think what happened is that when I had it as the last line, I did not have a line break at the end of the file, so when the line got sorted to the top, it got sorted without a \n, since as the final line, there was no \n. Neat little edge case.